### DISCUSSION TITLE
make the jukebox buttons bigger

### DISCUSSION THREAD

**Discussion by EpiCreeper** [Quaver/Quaver#4779] [Feature Request]
### Is your feature request related to a problem? Please describe.

the buttons at the jukebox menu are aligned exactly to the icon size, meaning when i try to press them half of the times i just change a point in the song
![image](https://github.com/Quaver/Quaver/assets/77532703/d87a04b0-1a99-4c95-b47d-29902f307690)

### Describe the solution you'd like

make the button detection bigger then the icons themselves.

**Comment by AiAe** [comment]
This is already changed in develop branch
